src/link.ts

```diff
@@ -486,7 +490,7 @@ export class LinkContainer {
     idTypeFilter : <(idtype: IDType, i: number, dataVis: IDataVis) => boolean>constantTrue,
```

Comment:
Ok. Why do you replace `constantTrue` sometimes and you keep it only in this two places?